src/main/resources/configCivcraft.yml

```diff
@@ -8998,7 +8983,7 @@ recipes:
     input:
       emeralds:
         material: EMERALD
-        amount: 5
+        amount: 25
```

Comment:
You can apply both prot and proj prot to the same item, which wasnt taken into account for the original price